### PR TITLE
Improvement for PR #2 - Title will be taken from the focused item

### PR DESCRIPTION
**Pull request by burekas7**
Improvement for the option of using the addon when kodi is not playing:
* Now the title will be taken from the focused item when it has details
(For example: Kodi Library, Exodus. But with Salts, Quasar, Specto this
will still not work until their developers will fix it)
* If it doesn't have details, it will remain "SearchFor" in order to
present "No Subtitles Found" result, so you can typing it manually in
the "Manual Search"

### CODE REVIEW

File: resources/lib/SUBUtilities.py [hunk 1]
```diff
@@ -69,6 +69,34 @@ def clean_title(item):
     item["title"] = re.sub(r'\([^\)]+\)\W*$', '', item["title"])
     item["tvshow"] = re.sub(r'\([^\)]+\)\W*$', '', item["tvshow"])
 
+def takeTitleFromFocusedItem():
+    try:
```
Comment (CaTzil) [inline resources/lib/SUBUtilities.py]: Why do you need try & catch?

File: resources/lib/SUBUtilities.py [hunk 2]
```diff
@@ -69,6 +69,34 @@ def clean_title(item):
...
+        labelType = xbmc.getInfoLabel("ListItem.DBTYPE")  #movie/tvshow/season/episode
+        labelMovieTitle = xbmc.getInfoLabel("ListItem.OriginalTitle")
+        labelYear = xbmc.getInfoLabel("ListItem.Year")
+        labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
+        labelSeason = xbmc.getInfoLabel("ListItem.Season")
+        labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
+
+        if labelType == 'movie':
```
Comment (CaTzil) [inline resources/lib/SUBUtilities.py]: The code can be minified to something simpler:
```python
title = 'SearchFor ...'
if labelType == 'movie' and labelMovieTitle and labelYear:
    title = labelMovieTitle + " " + labelYear
elif labelType == 'episode' and labelTVShowTitle and labelSeason and labelEpisode:
    title = ("%s S%.2dE%.2d" % (labelTVShowTitle, int(labelSeason), int(labelEpisode)))

return title
```

File: resources/lib/SUBUtilities.py (hunk 1 above)
Comment (burekas7) [inline resources/lib/SUBUtilities.py]: Basically you are right, there is no need for this actually.
I put it just to be sure if there will be kind of exception for some reason with one of the getInfoLable functions.

Comment (CaTzil) [inline resources/lib/SUBUtilities.py]: So you it`s just in case, remove it

**Comment by CaTzil** [comment]
Same thing should apply to subscenter plugin

File: resources/lib/SUBUtilities.py (hunk 2 above)
Comment (burekas7) [inline resources/lib/SUBUtilities.py]: Yes, you are right. As usual :)
I check it and it's working good. I will change it.

**Comment by burekas7** [comment]
Thank you.
All your comments will be implemented in the next PR.

**Pull request merged**